study.md

```diff
@@ -67,5 +67,11 @@ Starting with a client making a GET request to a particular URL, describe how
 the parts of Rails interact to produce and send a response.
 
 ```md
-<!-- your answer here -->
+1. Client sends requests to server
+2. Server sends requests to controller
```

Comment:
The router determines which controller action to take, based on the incoming URL and the verb.